docs/asciidoc/static/managing-multiline-events.asciidoc

```diff
@@ -0,0 +1,116 @@
+[[multiline]]
+=== Managing Multiline Events
+
+Several use cases generate events that span multiple lines of text. In order to correctly handle these multline events, 
+Logstash needs to know how to tell which lines are part of a single event.
+
+Whenever possible, avoid using multiline processing at all. When multiline processing is required, implement the
```

Comment:
"is best performed closest to the log source because that is where you have the strongest guarantees of ordered log processing"

Lost-ordering is why filter workers and far-from-the-source processing isn't advisable, so I wonder if we can describe the problem in that way. Something like "Doing multiline event processing requires your events be _in correct order_ as they appeared in the original log source" something something computers. ?